Header header
float64 pos_x
float64 pos_y
float64 estimated_pos_x
float64 estimated_pos_y
float64 amcl_diff
